docs/topics/circuit-breaker-mission-deploy-booster.adoc

```diff
@@ -8,9 +8,12 @@ include::booster-deploy-openshift-local.adoc[leveloffset=+1]
 
 include::oc-client-deploy-booster-download-note.adoc[leveloffset=+3]
 
-. Get your authentication token for using the `oc` CLI client with your {OpenShiftLocal} account. You can find this in the _Command Line Tools_ section your {OpenShiftLocal} Web console.
+. Get the command containing your authentication token for using the `oc` CLI client with your {OpenShiftLocal} account:
+.. Click on the question mark icon in the top right-hand corner of the Web console, next to your user name.
+.. Select _Command Line Tools_ in the drop-down menu.
+.. Click the button next to the first text box to copy the command containing the token.
```

Comment:
Does the text box have a label or something? "First text box" isn't very specific.

Reply:
@Ladicek unfortunately, the text box doesn't appear to have a usable label. There is a text headline, but it is more of an instruction than a label. (image below for reference)
<img width="975" alt="token" src="https://user-images.githubusercontent.com/367748/29895147-4976c8e8-8d95-11e7-954c-58dbf37afbe2.png"> 

I see two possible ways of working around this:
1.  We can provide an image for reference.
Admittedly rather clunky, because we have to update it if the UI changes.

2. We can provide a non-ambiguous descriptive identifier. 
Something like:

> The text box containing the login command with the hidden token.  


WDYT?

Reply:
Yea agree, "text box containing the `oc login ...` command with a hidden token" sounds good.